fv3core/tox.ini

```diff
@@ -0,0 +1,17 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py3
+
+[testenv:test]
+deps =
+    pytest
+    pytest-subtests
+    pytest-cov
+    -e external/fv3gfs-util
+    git+https://github.com/ai2cm/gt4py.git@v36#egg=gt4py
```

Comment:
Can you have the tag linked to the top level GT4PY_VERSION?

Reply:
if I can refactor the GT4PY_VERSION file to have `git+https://github.com/ai2cm/gt4py.git@v36#egg=gt4py` instead of `v36` then yes, otherwise this is a bit harder to accomplish.